lib/src/utils/ordinalLocale.dart

```diff
@@ -1,5 +1,6 @@
 List<String> localeOrdinals = [
   "en",
+  "id"
```

Comment:
Remove `id` from this file, `since it will push the other locales out of their place` or put it at the bottom and add its own ordinal string format like the other locales

Reply:
I pushed the review changes in newer commit 🙃
Thank you